akka-http-core/src/test/java/akka/http/javadsl/model/JavaApiTestCases.java

```diff
@@ -86,14 +86,14 @@ public static Object accessScalaDefinedJavadslContentTypeAndMediaType(ContentTyp
 
     // just for the sake of explicitly touching the interfaces
     if (mediaType.binary()) anything = (akka.http.javadsl.model.MediaType.Binary) mediaType;
-    if (1 == 2) anything = (akka.http.javadsl.model.MediaType.Multipart) mediaType;
```

Comment:
it's just a compile spec ;) @rkuhn is right that we don't need the ifs (was refactored from something that actually needed them)